src/apps/tools/OswAppKcalStats.cpp

```diff
@@ -0,0 +1,125 @@
+
+#include "./apps/tools/OswAppKcalStats.h"
+
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_hal.h>
+#include <osw_pins.h>
+
+#define MAX_VALUE 85
+#define MIN_VALUE 135
+
+// use queue
+void OswAppKcalStats::readyValue() {
+#if OSW_FEATURE_STATS_STEPS
```

Comment:
Temporary method for eliminating compilation errors.

Reply:
Could you post the compilation error here? I do not think that this should be permanent...

Reply:
See DEV_LUA https://open-smartwatch.github.io/resources/firmware/#support

Reply:
```
Run python3 .github/buildEverything.py -l cs-CZ -m LIGHT_EDITION_DEV_LUA
INFO:root:Setting language to cs-CZ...
INFO:root:Cleanup...
INFO:root:Compiling LIGHT_EDITION_DEV_LUA-cs-CZ-debug.bin...
ERROR:root:COMPILATION FAILED
ERROR:root:src/apps/tools/OswAppKcalStats.cpp: In member function 'void OswAppKcalStats::readyValue()':
src/apps/tools/OswAppKcalStats.cpp:22:38: error: 'class OswHal::Environment' has no member named 'getStepsOnDay'; did you mean 'getStepsToday'?
     weekValue[a] = hal->environment->getStepsOnDay(i);
                                      ^~~~~~~~~~~~~
                                      getStepsToday
*** [.pio/build/LIGHT_EDITION_DEV_LUA/src/apps/tools/OswAppKcalStats.cpp.o] Error 1
src/apps/tools/OswAppStepStats.cpp: In member function 'void OswAppStepStats::drawChart()':
src/apps/tools/OswAppStepStats.cpp:2[7](https://github.com/Open-Smartwatch/open-smartwatch-os/runs/6526851499?check_suite_focus=true#step:9:8):36: error: 'class OswHal::Environment' has no member named 'getStepsOnDay'; did you mean 'getStepsToday'?
     uint32_t s = hal->environment->getStepsOnDay(i);  // virtual step simulation
                                    ^~~~~~~~~~~~~
                                    getStepsToday
*** [.pio/build/LIGHT_EDITION_DEV_LUA/src/apps/tools/OswAppStepStats.cpp.o] Error 1
========================= [FAILED] Took 24.7[9](https://github.com/Open-Smartwatch/open-smartwatch-os/runs/6526851499?check_suite_focus=true#step:9:10) seconds =========================
Environment            Status    Duration
---------------------  --------  ------------
LIGHT_EDITION_DEV_LUA  FAILED    00:00:[24](https://github.com/Open-Smartwatch/open-smartwatch-os/runs/6526851499?check_suite_focus=true#step:9:25).790
==================== 1 failed, 0 succeeded in 00:00:24.790 ====================
```

Reply:
```
Run python3 .github/buildEverything.py -l cs-CZ -m LIGHT_EDITION_DEV_LUA
INFO:root:Setting language to cs-CZ...
INFO:root:Cleanup...
INFO:root:Compiling LIGHT_EDITION_DEV_LUA-cs-CZ-debug.bin...
ERROR:root:COMPILATION FAILED
ERROR:root:src/apps/tools/OswAppKcalStats.cpp: In member function 'void OswAppKcalStats::readyValue()':
src/apps/tools/OswAppKcalStats.cpp:22:38: error: 'class OswHal::Environment' has no member named 'getStepsOnDay'; did you mean 'getStepsToday'?
     weekValue[a] = hal->environment->getStepsOnDay(i);
                                      ^~~~~~~~~~~~~
                                      getStepsToday
*** [.pio/build/LIGHT_EDITION_DEV_LUA/src/apps/tools/OswAppKcalStats.cpp.o] Error 1
src/apps/tools/OswAppStepStats.cpp: In member function 'void OswAppStepStats::drawChart()':
src/apps/tools/OswAppStepStats.cpp:2[7](https://github.com/Open-Smartwatch/open-smartwatch-os/runs/6526851499?check_suite_focus=true#step:9:8):36: error: 'class OswHal::Environment' has no member named 'getStepsOnDay'; did you mean 'getStepsToday'?
     uint32_t s = hal->environment->getStepsOnDay(i);  // virtual step simulation
                                    ^~~~~~~~~~~~~
                                    getStepsToday
*** [.pio/build/LIGHT_EDITION_DEV_LUA/src/apps/tools/OswAppStepStats.cpp.o] Error 1
========================= [FAILED] Took 24.7[9](https://github.com/Open-Smartwatch/open-smartwatch-os/runs/6526851499?check_suite_focus=true#step:9:10) seconds =========================
Environment            Status    Duration
---------------------  --------  ------------
LIGHT_EDITION_DEV_LUA  FAILED    00:00:[24](https://github.com/Open-Smartwatch/open-smartwatch-os/runs/6526851499?check_suite_focus=true#step:9:25).790
==================== 1 failed, 0 succeeded in 00:00:24.790 ====================
```

Reply:
https://github.com/Open-Smartwatch/open-smartwatch-os/runs/6526851499?check_suite_focus=true